docs/components/modeler/forms/form-element-library/forms-element-library-textarea.md

```diff
@@ -1,7 +1,7 @@
 ---
 id: forms-element-library-textarea
 title: Text area
-description: Learn about the library textarea form element to read and edit multiline textual data.
+description: Learn about the textarea form element to read and edit multiline textual data.
 ---
 
 A text area allowing the user to read and edit multiline textual data.
```

Comment:
Is it "textarea" after the HTML element `<textarea>`? I see a discrepancy between the description and the first line.

Reply:
textarea vs. text area

Reply:
@Skaiir @pinussilvestrus do you have thoughts here? How is this referenced in the UI? Happy to clarify and add into the glossary.

Reply:
It's indeed a `textarea` HTML element in the end, but we call it `text area` in the UI.

Reply:
The docs should be consistent with the UI here I think.

Reply:
Updated the description 👍